ttbd/ttbl/ui.py

```diff
@@ -514,6 +514,45 @@ def _target_button_get(target: ttbl.test_target, inventory: dict, kws: dict):
     return buttons_component_description
 
 
+
+def _target_tunnel_collect(target, inventory, state):
+    state['server_ips'] = commonl.local_ip_addrs_cached()
+    state['target_ips'] = {}
+    boot_ic_name = inventory.get('pos', {}).get(
```

Comment:
maybe we could add a small example of the json for `pos`. Just to make it easier to understand the structure.